api/server/runner_fninvoke_test.go

```diff
@@ -374,7 +374,8 @@ func TestInvokeRunnerMinimalConcurrentHotSync(t *testing.T) {
 		expectedCode    int
 		expectedHeaders map[string][]string
 	}{
-		{"/invoke/fn_id", `{"sleepTime": 100, "isDebug": true}`, "POST", http.StatusOK, nil},
+		{"/invoke/fn_id", `{"sleepTime": 100, "isDebug": true}`, http.MethodPost, http.StatusOK, nil},
+		{"/invoke/fn_id", `{"sleepTime": 100, "isDebug": true}`, http.MethodPut, http.StatusMethodNotAllowed, nil},
```

Comment:
could you move this to the TestFnInvokeRunnerExecution block please? this test is for concurrency and testing the http method has nothing to do with testing concurrency, and could test the actual error message from there too.

Reply:
Yup, thanks, addressed.